docs/index.md

```diff
@@ -9,22 +9,28 @@ New to Presidio? Read this material to quickly get up and running.
 ## Table of Contents
 
 General:
+
+- [Product Overview](overview.md)
+- [API Samples](samples.md)
 - [How to deploy presidio](deploy.md)
 - [Supported field types](field_types.md)
 - [Database and storage scanner](tutorial_scheduler.md)
 - [Architecture](design.md)
 
 Development:
+
 - [Setting up a development environment](development.md)
 - [Adding custom fields](custom_fields.md)
 - [Best practices for developing new recognizers](developing_recognizers.md)
 - [Analyzer service tutorial](tutorial_analyzer.md)
 - [Calling the different services](tutorial_service.md)
```

Comment:
Could you please the new tutorial_connector.md to the list? https://github.com/microsoft/presidio/blob/master/docs/tutorial_connector.md

Reply:
done